docs/user-guide/composer.md

```diff
@@ -1,3 +1,45 @@
-# Composer
 
-TODO
+ This guide will walk you through the process of using our email composition tool effectively.
+ ###  Creating a New Email in web
+1. Click on the "New Email" button.
+2. A new email composer will appear.
+3. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them
```

Comment:
```suggestion
3. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them. The identities you can use in the `From` field can be edited on the [profile page](profile.md).
```